lumibot/strategies/_strategy.py

```diff
@@ -63,6 +65,8 @@ def __init__(
         account_history_db_connection_str=None,
```

Comment:
I think we should rename this variable/add a new one with a different name because it's not just for account history anymore

Reply:
We could make it simply db_connection_str? I thought about doing this but it would break quite a lot of current strategies. This could be included in the next major release perhaps?

Reply:
i think we should keep both for backwards compatibility and throw a warning if the old one is used. db_connection_str makes a lot of sense as a name. I would keep account_history_db_connection_str in init but rename it everywhere else to db_connection_str, and something like this
`if account_history_db_connection_str: 
db_connection_str = account_history_db_connection_str
logging.warning("account_history_db_connection_str is deprecated and will be removed in future versions, please use db_connection_str instead")
`